docs/setup-cluster/deploy-cluster/slurm/hpc-security-considerations.rst

```diff
@@ -0,0 +1,68 @@
+.. _hpc-security-considerations:
+
+######################################
+ HPC Launcher Security Considerations
+######################################
+
+User authentication in Determined is enforced using :ref:`users`.  Determined sends requests on behalf of those authenticated users to the HPC launcher 
```

Comment:
I "think" that a comma is needed after "launcher", because if we were to speak this sentence verbally, we would likely pause after we say "launcher".